README.md

```diff
@@ -130,18 +122,26 @@ From time to time, we'll be updating the Starfish code. You can get the newest c
 
 # Other Important Info
 
-This tool by default checks for CommitCommentEvents, IssueCommentEvents, IssuesEvents, PullRequestEvents, PullRequestReviewEvents, and PullRequestReviewCommentEvents. We do not look for PushEvents because those are usually used for personal projects, not actual open source contributions.
+-   Caveats: [The GitHub Rest API](https://docs.github.com/en/free-pro-team@latest/rest) only holds the most recent 300 events for each user. Also, events older than 90 days will not be included (even if the total number of events in the timeline is less than 300). So, if you're looking for contributions from 4 months ago, Starfish won't be able to find any. And if you are looking for contributions from 2 months ago, and one or more of your users is very active (300 events or more per month!), your result might not be completely accurate.
 
-You can also override the list of events to check by editing the `GITHUB_IMPORTANT_EVENTS` variable in your ".env" file with a comma-separated list of events
+-   Also, we know that there are many types of contributions to open source - not just code, and not just on GitHub. At Indeed, we have a Google form Indeedians can fill out to tell us about other contributions they've made. We recommend you do that as well.
 
-Starfish supports filtering events based on the specific action taken. For example, you might want to count when a pull request is closed, but not when it is opened. To do that, the list of important events can include types (like "PullRequestEvent") or a specific action of a type (like "PullRequestEvent.closed").
-You can list multiple actions for the same event type. Visit [GitHub event types](https://docs.github.com/en/free-pro-team@latest/developers/webhooks-and-events/github-event-types#event-object-common-properties) for more information.
+-   Lastly, if you're using Starfish we'd ***love*** to hear about it. What are you using Starfish for? Does it work well for you? You can leave us a comment over in the [Discussions](https://github.com/indeedeng/starfish/discussions).
 
-Caveats: [The GitHub Rest API](https://docs.github.com/en/free-pro-team@latest/rest) only holds the most recent 300 events for each user. Also, events older than 90 days will not be included (even if the total number of events in the timeline is less than 300). So, if you're looking for contributions from 4 months ago, Starfish won't be able to find any. And if you are looking for contributions from 2 months ago, and one or more of your users is very active (300 events or more per month!), your result might not be completely accurate.
+# Troubleshooting and Changelog
```

Comment:
It feels a bit weird to me that the heading lists troubleshooting first, but the changelog section comes first.

Reply:
I'll flip that.
(I have a logical reason why it makes sense this way, in my head, but you're totally right that anyone who's not me reading it would think it's strange.)